src/Generator.cpp

```diff
@@ -1,25 +1,18 @@
 #include "Generator.hpp"
 
-Lines getAllVectorPermutation_(int vSize)
+bool nextPermutation(Line &row)
```

Comment:
Please keep existing style where &, * is sticked to Type. 
Line& row <--- 
this convention is used on master in Board.cpp

Please change it also below.

Reply:
done